From cf0495159a12ca5aa146b625b0083681e9636932 Mon Sep 17 00:00:00 2001
From: Sorin Sbarnea <ssbarnea@redhat.com>
Date: Wed, 12 May 2021 14:31:25 +0100
Subject: [PATCH] Remove direct dependency on markupsafe

This sorts potential dependency conflicts with jinja2 which is the
only place where markupsafe is used (since version 2.7)
---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index f36f57be..124dc40c 100644
--- a/setup.py
+++ b/setup.py
@@ -16,7 +16,6 @@
     'python-slugify>=4.0.0',
     'requests>=2.23.0',
     'six>=1.10',
-    'MarkupSafe<2.0.0',
 ]
 
 setup(
